src/org/zaproxy/zap/GuiBootstrap.java

```diff
@@ -345,15 +340,37 @@ private static void setDefaultViewLocale(Locale locale) {
      * {@link java.net.InetAddress InetAddress}) preventing some ZAP options from being correctly applied.
      */
     private void setupLookAndFeel() {
-        if (lookAndFeelSet) {
-            return;
+    	
+    	String lookAndFeelClassnameFromCommandline = System.getProperty("swing.defaultlaf");
+    	OptionsParam options = Model.getSingleton().getOptionsParam();
+    	String lookAndFeelnameFromOptions = options.getViewParam().getLookAndFeel();
+    	String lookAndFeelClassnameFromOptions = "";
+    	UIManager.LookAndFeelInfo[] looks = UIManager.getInstalledLookAndFeels();
+    	
+    	for(UIManager.LookAndFeelInfo index : looks) {
```

Comment:
- `look` instead of `index`?
 - It can be extracted a method (doing the same as the `Nimbus` check done later).